example/README.md

```diff
@@ -77,3 +77,28 @@ Cleanup:
 ```sh
 docker compose -f docker-compose-splunk.yaml down
 ```
+
+### Direct to Splunk Observability Cloud
+
+Run the instrumented application:
+
+```sh
+export OTEL_SERVICE_NAME="splunk-otel-go-example"
+export OTEL_RESOURCE_ATTRIBUTES="deployment.environment=$(whoami)"
+SPLUNK_REALM=<realm> SPLUNK_ACCESS_TOKEN=<access_token> go run .
+```
+
+You can find the collected telemetry in:
+
+- Splunk Observability Cloud: <https://app.signalfx.com/#/apm?environments=YOURUSERNAME>
+  > Note: Processing might take some time.
+
+### FIPS mode - Linux
```

Comment:
Do we need to say something about stability guarantees here?

https://go.dev/src/crypto/internal/boring/README

> We have been working inside Google on a fork of Go that uses BoringCrypto (the core of [BoringSSL](https://boringssl.googlesource.com/boringssl/)) for various crypto primitives, in furtherance of some work related to FIPS 140. We have heard that some external users of Go would be interested in this code as well, so we have published this code here in the main Go repository behind the setting GOEXPERIMENT=boringcrypto.
> 
> Use of GOEXPERIMENT=boringcrypto outside Google is unsupported. This mode is not part of the [Go 1 compatibility rules](https://go.dev/doc/go1compat), and it may change incompatibly or break in other ways at any time.

Reply:
I think we should indeed add a notice. Also probably that using it does not implies that something is FIPS compliant.